From beb7a7ebf914971bbc5ca06c481e87e39dfe81d8 Mon Sep 17 00:00:00 2001
From: Jordan <jordan@smartdust.me>
Date: Thu, 15 Jun 2023 08:11:56 +0200
Subject: [PATCH 1/2] patch to set constant room id

---
 src/core/room.go | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/core/room.go b/src/core/room.go
index b057f21..1ea656a 100644
--- a/src/core/room.go
+++ b/src/core/room.go
@@ -22,6 +22,7 @@ type StreamSession struct {
 }
 
 var roomMap = make(map[string]*Room)
+const labskRoomId = "labsk"
 
 func GetRoom(id string) *Room {
     return roomMap[id]
@@ -38,10 +39,7 @@ func NewRoom(callerConn *websocket.Conn) *Room {
 }
 
 func newRoomID() string {
-    id := GetRandomName(0)
-    for GetRoom(id) != nil {
-        id = GetRandomName(0)
-    }
+    id := labskRoomId;
     return id
 }
 
-- 
2.34.1

